curriculum/challenges/english/02-javascript-algorithms-and-data-structures/object-oriented-programming/inherit-behaviors-from-a-supertype.english.md

```diff
@@ -81,13 +81,10 @@ Animal.prototype = {
   }
 };
 
-// Add your code below this line
+// Only change code below this line
 
 let duck; // Change this line
 let beagle; // Change this line
-
-duck.eat(); // Should print "nom nom nom"
-beagle.eat(); // Should print "nom nom nom"
```

Comment:
Do we maybe want to leave these lines in? Yes, the user can write them themselves but I don't see why we should remove these

Reply:
That is the point.  The user can write these and we do not have to translate the comment.  At this point in the curriculum, the user has already learned how to use `console.log`, so this would encourage debugging code.

Reply:
Do you want me to reference that in #38040?

Reply:
@thecodingaviator No, let's create a separate issue to discuss it.